src/i18n/en.js

```diff
@@ -215,6 +215,32 @@ const messages_en = {
       transfer: 'Bank transfer',
       valid_thru: 'valid thru',
     },
+    purchase_summary: {
+      buy_button: 'BUY',
+      discount_for_payment_paid: 'Positive balance:',
+      discount_for_prepayment: 'Discount for prepayment:',
+      edit_add_recipients_button: 'Edit or add recipients',
+      error_message: 'Your payment couldn’t be processed. Choose a different payment method or try it later.',
+      explanatory_legend: 'The renewal is automatic and you can cancel it whenever you want. The price of the Plan does not include taxes.',
+      header: 'Purchase summary',
+      month_label: ' month',
+      months_label: ' months',
```

Comment:
You could instead of having to variables handle the plurals like [here](https://github.com/FromDoppler/doppler-webapp/blob/master/src/i18n/en.js#L572)